src/directory/directory.mjs

```diff
@@ -7,24 +7,15 @@ export const directory = {
     {
       path: 'src/pages/[platform]/index.tsx',
       children: [
-        {
-          path: 'src/pages/[platform]/how-amplify-works/index.mdx'
```

Comment:
Do we expect this page to go away?

Reply:
Oh I see, it moved down. Do we want it down there @heatheramz?

Reply:
https://github.com/aws-amplify/docs/pull/6225/files#diff-782c64a4a6ba1f0bff31bf639532b0169bd826b3e4208dc69ed6af06a98b08aeR256